recovery/src/main/java/com/kin/ecosystem/recovery/backup/view/BackupActivity.java

```diff
@@ -48,6 +60,8 @@ public void startBackupFlow() {
 
 		if (backupInfoFragment == null) {
 			backupInfoFragment = BackupInfoFragment.newInstance(backupPresenter);
+		} else {
```

Comment:
Our entire usage of newInstance is broken - new instance design pattern will only work for priorities that can be  stored in bundle using setArguments. just setting properties will not be store if fragment will be recreated